.gitpod/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-java-11
+
+RUN bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
+    sdk install java 11.0.9.j9-adpt && \
```

Comment:
why use j9 here and not adoptium which we use everywhere else ?

Reply:
oh ! I didn't see that. After reading the project, I didn't find the image to use. Where I can find it please ?